src/missed_spans.rs

```diff
@@ -100,6 +100,10 @@ impl<'a> FmtVisitor<'a> {
         }
 
         if snippet.trim().is_empty() && !out_of_file_lines_range!(self, span) {
+            if self.is_start_span(span) {
```

Comment:
Hmm, why do we need this?

Reply:
This is used for the files read from attribute(ex. imp-a.rs or imp-b.rs).
These file span don't start from 0, start from the BytePos of the end of the file written attribute(in this case lib.rs). So we need to implement the logic to avoid the newline.

Reply:
I see, thank you for the explanation! In that case, could you merge this `if` condition with the one above to make the intent more clear? E.g., something like the following:

```rust
        // Do nothing for spaces in the beginning of the file
        let is_start_span = (start == BytePos(0) && end.0 as usize == snippet.len()) || self.is_start_span(span);
        if is_start_span && snippet.trim().is_empty() {
            return;
        }
```

Reply:
Certainly. Thank you for the review. I fixed it.